src/Pagination.jsx

```diff
@@ -334,8 +336,11 @@ class Pagination extends React.Component {
     let lastPager = null;
     let gotoButton = null;
 
+    const { pagerCount } = props;
+    const boundaryRemainder = pagerCount % 2 === 0 ? 1 : 0;
     const goButton = (props.showQuickJumper && props.showQuickJumper.goButton);
-    const pageBufferSize = props.showLessItems ? 1 : 2;
+    const halfPagerCount = Math.max(1, Math.floor((pagerCount - 1) / 2));
+    const pageBufferSize = props.showLessItems ? 1 : halfPagerCount;
```

Comment:
@afc163 ，这么看起来 `showLessItems` 是不是可以提示 deprecated 了？

Reply:
对，应该是重复的。

Reply:
@liuchuzhang ，如果提供了 `pagerCount` 优先以 `pagerCount` 为准。当用户用了 `showLessItems` 时提示用户 '`showLessItems` is deprecated since 1.18.0. Please use `pagerCount` instead.'